### PR TITLE
Add ernie gram 

### PR DESCRIPTION
**Pull request by Steffy-zxf**
### PR types

 New features 

### PR changes

Models 

### Description

* add ernie-gram-zh pretrained models
* ernie-gram-zh can do text cls, seq-cls, Q&A tasks and so on

### CODE REVIEW

File: paddlenlp/transformers/ernie_gram/matching_param_name.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                                     static_params_dir,
+                                     static_to_dygraph_param_name,
+                                     model_name='static'):
+    files = os.listdir(static_params_dir)
+
+    state_dict = {}
+    model_name = model_name
+    for name in files:
+        path = os.path.join(static_params_dir, name)
+        # static_para_name = name.replace('@HUB_chinese-roberta-wwm-ext-large@',
```
Comment (ZeyuChen) [inline paddlenlp/transformers/ernie_gram/matching_param_name.py]: 删除。

File: paddlenlp/transformers/ernie_gram/tokenizer.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            The special token for cls. 
+            Defaults to "[CLS]".
+        mask_token (str, optional): 
+            The special token for mask.
+            Defaults to "[MASK]".
+    
+    Examples:
+        .. code-block:: python
+            from paddlenlp.transformers import ErnieTokenizer
+            tokenizer = ErnieTokenizer.from_pretrained('ernie-1.0')
```
Comment (ZeyuChen) [inline paddlenlp/transformers/ernie_gram/tokenizer.py]: 样例同步更新下。

File: paddlenlp/transformers/ernie_gram/matching_param_name.py (hunk 1 above)
Comment (Steffy-zxf) [reply paddlenlp/transformers/ernie_gram/matching_param_name.py]: done

File: paddlenlp/transformers/ernie_gram/tokenizer.py (hunk 2 above)
Comment (Steffy-zxf) [reply paddlenlp/transformers/ernie_gram/tokenizer.py]: done

**Review by wawltor — APPROVED** [review]
LGTM

**Pull request merged**